__tests__/formats/all.test.js

```diff
@@ -26,9 +26,34 @@ var file = {
 };
 
 var dictionary = {
+  "allProperties": [{
+      value: '#FF0000',
+      original: { value: '#FF0000' },
+      name: 'color_red',
+      attributes: {
+         category: 'color',
+         type: 'red',
+         item: undefined,
+         subitem: undefined,
+         state: undefined
+      },
+      path: ['color','red']
+  }],
   "properties": {
     "color": {
-      "red": {"value": "#FF0000"}
+      "red": {
+        value: '#FF0000',
+        original: { value: '#FF0000' },
+        name: 'color_red',
+        attributes: {
+           category: 'color',
+           type: 'red',
+           item: undefined,
+           subitem: undefined,
+           state: undefined
+        },
+        path: ['color','red']
+      }
     }
```

Comment:
Was this change correct?

Reply:
Kind of.. we will need to update our snapshots, which is fine. We should probably discuss how to better test these formats because it seems a bit weird test all formats against the same dictionary. For example, the output of the format might not make sense with this input. If you look at the snapshots, some have 'NaN' in them or code that would not actually compile. We might want to think about having a different test and snapshot for each format. Maybe we can save that for later. For now we could just update the snapshots, you can do that in Jest's interactive mode when it comes across a failing snapshot test you can update the snapshot

Reply:
Updated the snapshots. Maybe we should add an issue for your recommendation?